docs/pages/7_0_0_release_notes.md

```diff
@@ -22,6 +22,8 @@ This is a {{ site.pmd.release_type }} release.
 ### Fixed Issues
 
 ### API Changes
+*   [#1648](https://github.com/pmd/pmd/pull/1702): \[apex,vf] Remove CodeClimate dependency - [Robert Sösemann](https://github.com/rsoesemann)
```

Comment:
The fact that some properties were removed and their names should be mentioned here, probably with a short mention of the reason. The link to the issue can be mentioned for more details, but we try to explain how users may be affected by API changes in the release notes directly, without having to open an external link to find out what changed.